src/serlio/modifiers/RuleAttributes.h

```diff
@@ -40,12 +40,14 @@ constexpr const wchar_t* ANNOT_DIR = L"@Directory";
 constexpr const wchar_t* ANNOT_FILE = L"@File";
 constexpr const wchar_t* ANNOT_ORDER = L"@Order";
 constexpr const wchar_t* ANNOT_GROUP = L"@Group";
+constexpr const wchar_t* ANNOT_IMPORTS = L"_$IMPORTS";
+constexpr const wchar_t* ANNOT_IMPORTS_KEY = L"fullPrefix";
 
 constexpr int ORDER_FIRST = std::numeric_limits<int>::min();
 constexpr int ORDER_NONE = std::numeric_limits<int>::max();
 
 using AttributeGroup = std::vector<std::wstring>;
-using AttributeGroupOrder = std::map<AttributeGroup, int>;
+using AttributeGroupOrder = std::map<std::tuple<std::wstring,AttributeGroup>, int>;
```

Comment:
Consider using a std::pair if the tuple only has two items, leads to slightly more compact code.